hpx/components/containers/partitioned_vector/partitioned_vector.hpp

```diff
@@ -732,6 +732,18 @@ namespace hpx
         }
 
     public:
+
+        std::vector< hpx::id_type > get_partitions_ids()
+        {
+            std::vector< hpx::id_type > ids;
+
+            for(auto const part_data : partitions_)
```

Comment:
It allows my array_view to get all it needs to return the data of a partition. Saving the ids is costless as well.